src/tof/chopper.py

```diff
@@ -198,27 +197,21 @@ class ChopperReading(ComponentReading):
     open_times: sc.Variable
     close_times: sc.Variable
     data: sc.DataArray
-    toas: ReadingField
-    wavelengths: ReadingField
-    birth_times: ReadingField
-    speeds: ReadingField
+
+    def _repr_stats(self) -> str:
+        return (
+            f"visible={int(self.data.sum().value)}, "
+            f"blocked={int(self.data.masks['blocked_by_me'].sum().value)}"
+        )
 
     def __repr__(self) -> str:
         out = f"ChopperReading: '{self.name}'\n"
         out += f"  distance: {self.distance:c}\n"
         out += f"  frequency: {self.frequency:c}\n"
         out += f"  phase: {self.phase:c}\n"
         out += f"  cutouts: {len(self.open)}\n"
-        out += "\n".join(
-            f"  {key}: {getattr(self, key)}"
-            for key in ('toas', 'wavelengths', 'birth_times', 'speeds')
-        )
-        return out
+        out += "  fields: toa, wavelength, birth_time, speed\n  "
```

Comment:
What is this line for? `ChopperReading` doesn't have these fields anymore.

Reply:
It should be getting them from [`ComponentReading`](https://github.com/scipp/tof/pull/76/files#diff-de178afe47cb70529e3231c850f15bc7c3b9f10477c1391dd5086e87ba1394d3R78)

It was just to add info in the repr as to what was available as data access, but it doesn't bring much else. We can also remove it. If we change/add/remove some fields in `ComponentReading`, this could get out of sync if we forget to also update here. I tried finding an automatic way of adding this but none of it was pretty...

Reply:
Oh, I missed that the chopper class inherits from that.

I'd remove the line, it doesn't represent the contents of the object, but is a minimal form of the `help` function.